packages/core/src/domain/ai-brief/modify-resources.ts

```diff
@@ -476,15 +509,21 @@ export type SlimCondition = Omit<Condition, "name" | "category" | "clinicalStatu
 
 function getSlimCondition(res: Condition): SlimCondition {
   const updRes = cloneDeep(res);
+  const name = getUniqueDisplaysString(updRes.code);
+  const category = getUniqueDisplaysString(updRes.category);
+  const clinicalStatus = isUselessStatus(getUniqueDisplaysString(updRes.clinicalStatus))
+    ? undefined
+    : getUniqueDisplaysString(updRes.clinicalStatus);
+
   delete updRes.code;
+  delete updRes.category;
+  delete updRes.clinicalStatus;
```

Comment:
Since we're overriding those when we build the object, we don't need to delete them here. But it doesn't break anything, mostly to minimize code.